__tests__/func.test.js

```diff
@@ -0,0 +1,8 @@
+const { createRefObj } = require("../func/func");
```

Comment:
I don't think this code belongs on this branch/PR - should be removed and put on the correct branch